completion/completion.plugin.zsh

```diff
@@ -6,8 +6,12 @@ if [[ "$TERM" == 'dumb' ]]; then
   return 1
 fi
 
+#
+# Config
```

Comment:
I wrote this in contributing ages ago - https://github.com/belak/zsh-utils/blob/main/CONTRIBUTING.md#sections

I'm not sure it's accurate any more, but I would like to be consistent. I think that's why I left it under "Requirements" initially.

Reply:
I'll be honest - I missed that part in the contributing guidelines. I have no strong preference here, so I can remove that.